src/main/java/com/minecolonies/coremod/colony/buildings/AbstractBuildingWorker.java

```diff
@@ -463,11 +463,6 @@ public void onWorldTick(@NotNull final TickEvent.WorldTickEvent event)
     {
         super.onWorldTick(event);
 
-        if (event.phase != TickEvent.Phase.END)
-        {
-            return;
-        }
-
         // If we have no active worker, grab one from the Colony
         // TODO Maybe the Colony should assign jobs out, instead?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)